build/ci/unit-tests-template.yml

```diff
@@ -0,0 +1,48 @@
+# Template used by unit-tests.yml
+
+jobs:
+- job: ${{ parameters.jobName }} (${{ parameters.buildConfiguration }})
+  pool:
+    name: ${{ parameters.poolName }}
+    queue: ${{ parameters.poolQueue }}
+  timeoutInMinutes: 20
+  steps:
+    - script: $(Build.SourcesDirectory)\build.cmd /build /test /ci /sign /diagnostic /clearnugetcache /no-deploy /no-integration /no-ibc /configuration ${{ parameters.buildConfiguration }}
+    - task: PublishBuildArtifacts@1
+      inputs:
+        PathtoPublish: '$(Build.SourcesDirectory)\artifacts\${{ parameters.buildConfiguration }}\log'
+        ArtifactName: '${{ parameters.buildConfiguration }} unit test logs'
+        publishLocation: Container
+      continueOnError: true
+      condition: failed()
+    - task: PublishBuildArtifacts@1
+      inputs:
+        PathtoPublish: '$(Build.SourcesDirectory)\artifacts\${{ parameters.buildConfiguration }}\bin'
+        ArtifactName: '${{ parameters.buildConfiguration }} bin folder'
+        publishLocation: Container
+      continueOnError: true
+      condition: failed()
+    - task: PublishBuildArtifacts@1
+      inputs:
+        PathtoPublish: '$(Build.SourcesDirectory)\artifacts\${{ parameters.buildConfiguration }}\VSSetup'
+        ArtifactName: '${{ parameters.buildConfiguration }} VSSetup folder'
+        publishLocation: Container
+      continueOnError: true
+      condition: failed()
+    - task: PublishBuildArtifacts@1
+      inputs:
+        PathtoPublish: '$(Build.SourcesDirectory)\artifacts\${{ parameters.buildConfiguration }}\TestResults'
+        ArtifactName: '${{ parameters.buildConfiguration }} Test Result Logs'
+        publishLocation: Container
+      continueOnError: true
+      condition: failed()
+    - task: PublishTestResults@2
+      inputs:
+        testRunner: 'xUnit'	
+        testResultsFiles: '**/*.xml' 	
+        searchFolder: '$(Build.SourcesDirectory)\artifacts\${{ parameters.buildConfiguration }}\TestResults'	
+        configuration: '${{ parameters.buildConfiguration }}'	
+        publishRunAttachments: true	
+        failTaskOnFailedTests: true
```

Comment:
This used to not be specified for the Spanish build.

Reply:
It was an oversight.